proposed/2023/pmui-tab-consolidation.md

```diff
@@ -0,0 +1,71 @@
+# NuGet PMUI Tab Consolidation
+
+- Allie Barry @albarry4
+- Start Date 7/27/2023
+
+## Summary
+
+This proposal suggests a solution to reduce the 3 tab format of the NuGet PMUI in Visual Studio as a first step to simplify and modernize the UI, and to address recent customer feedback.  
+
+## Motivation 
+
+We have recently recieved feedback from custmers that the NuGet PMUI design is "clunky" and "requires too much clicking over all the tabs, checkboxes... takes too much clicking and surfing to navigate to varius changelogs". Based on this feedback, I wanted to propose an initial solution to specifcially target and reduce the amount of clicking, and to simplify and consolidate the controls within the PMUI by reducting the number of tabs available. Specifically, this proposal aims to consolidate the "Installed" and "Updates" tabs into a single window titled "My Packages".
+
+Note: I want to include data here about tab click-through rates, but am still working on getting access to that data!
+
+## Explanation
+
+### Functional explanation
+
+Put simply, the end goal of this proposal is to reduce the number of tabs in the NuGet PMUI, and to simplify the format of the User interface. 
+
+In this new experience, instead of seeing three tabs ("Browse", "Installed", "Update"), the user will see two tabs at the top of the screen, "Browse", and "My Packages". Below, I will describe how these two tabs will behave. 
+
+The "Browse" tab will remain unchanged from the current design. 
+
+Within the "My Packages" tab, the user will see a list of the packages that they have installed in the current project. The layout of the tab will remain mostly unchanged from the general layout of the seperate "Installed" and "Updates" Tabs that we see today, but with a few tweaks to consolidate all of this information into one tab. 
+
+If no installed backages have an update available, this is what the user will see:
+
+![image](https://github.com/albarry4/Home/assets/89422562/ce769bc6-ae6d-48b0-b093-730881d7ef2d)
+
+
+If the user has a package installed with an update available, this is what the user will see: 
```

Comment:
I like how this looks if my goal is to look at the packages that have updates only.
Should this be the default though? Or do we want a filter/checkbox something that does this separation. 

Is there a world where the packages with updates/without updates are within a single list?